controllers/suite_test.go

```diff
@@ -263,3 +277,10 @@ func buildCluster(opts opniClusterOpts) *v1beta1.OpniCluster {
 		},
 	}
 }
+
+func generateSHAID(name string, namespace string) string {
+	hash := sha1.New()
+	hash.Write([]byte(name + namespace))
+	sum := hash.Sum(nil)
+	return fmt.Sprintf("%x", sum[:3])
```

Comment:
`hex.EncodeToString` probably better here (although it doesnt really make a difference)